src/python/pants/python/python_setup.py

```diff
@@ -274,17 +279,38 @@ def get_pex_python_paths():
             return []
 
     @staticmethod
-    def get_pyenv_paths(pyenv_root_func=None):
+    def get_pyenv_paths(pyenv_root_func=None, pyenv_local=False):
         """Returns a list of paths to Python interpreters managed by pyenv.
 
         :param pyenv_root_func: A no-arg function that returns the pyenv root. Defaults to
                                 running `pyenv root`, but can be overridden for testing.
+        :param bool pyenv_local: If True, only use the interpreter specified by
+                                 '.python-version' file under `build_root`. Default to False.
```

Comment:
Awesome. No need for the `Default to False` part, though, as the code already shows that.